src/common/cluster/authorization-namespace-review.injectable.ts

```diff
@@ -0,0 +1,56 @@
+/**
+ * Copyright (c) OpenLens Authors. All rights reserved.
+ * Licensed under MIT License. See LICENSE in root directory for more information.
+ */
+
+import type { KubeConfig } from "@kubernetes/client-node";
+import { AuthorizationV1Api } from "@kubernetes/client-node";
+import logger from "../logger";
```

Comment:
logger shouldn't be imported globally in new code, you can get the instance from `di.inject(loggerInejctable)` within the `instantiate` function below.

Reply:
this was copied from src/common/cluster/authorization-review.injectable.ts

I changed it in both files